guides/doc-Planning_for_Project/topics/Deployment_Scenarios.adoc

```diff
@@ -34,8 +34,8 @@ ifdef::satellite[]
 
 In high security environments where hosts are required to function in a closed network disconnected from the Internet, {ProjectName} can provision systems with the latest security updates, errata, packages and other content.
 In such case, {ProjectServer} does not have direct access to the Internet, but the layout of other infrastructure components is not affected.
-For information about installing {ProjectServer} from a disconnected network, see {MultiBaseURL}installing_satellite_server_from_a_disconnected_network/[Installing {ProjectServer} from a Disconnected Network].
-For information about upgrading a disconnected {Project}, see {MultiBaseURL}upgrading_and_updating_red_hat_satellite/upgrading_red_hat_satellite#upgrading_a_disconnected_{project-context}[Upgrading a Disconnected {ProjectServer}] in _Upgrading and Updating {ProjectName}_.
+For information about installing {ProjectServer} from a disconnected network, see {InstallingDisconnectedDocURL}[{InstallingServerDisconnectedDocTitle}].
+For information about upgrading a disconnected {Project}, see {UpgradingDocURL}#upgrading_a_disconnected_{project-context}[Upgrading a Disconnected {ProjectServer}] in _{UpgradingDocTitle}_.
```

Comment:
This link has two hashes (`index##`), I don't know whether it is an issue or not.